scalabel/label/typing.py

```diff
@@ -94,3 +93,37 @@ def __init__(self, **data: Any) -> None:  # type: ignore
         if "name" in data:
             data["name"] = str(data["name"])
         super().__init__(**data)
+
+
+class Category(BaseModel):
+    """Define Scalabel label attributes."""
+
+    name: str
+    subcategories: Optional[List["Category"]]
+
+
+Category.update_forward_refs()
+
+
+class Attribute(BaseModel):
+    """Define Scalabel category type."""
+
+    name: str
+    toolType: str
+    tagText: str
+    values: Optional[List[str]]
+
+
+class Config(BaseModel):
+    """Define metadata of the dataset."""
+
+    resolution: Optional[Tuple[int, int]]
```

Comment:
a `Size` class will be easier to understand here.

Reply:
done